assets/javascripts/discourse/templates/admin-plugins-gamification.hbs

```diff
@@ -72,48 +101,81 @@
-      class="btn-primary"
-      action=(action (mut creatingNew) true)
-    }}
+    {{#if model.leaderboards}}
+      {{d-button
+        label="gamification.leaderboard.new"
+        title="gamification.leaderboard.new"
+        icon="plus"
+        class="btn-primary leaderboard-admin__btn-new"
+        action=(action (mut creatingNew) true)
+      }}
+    {{/if}}
   {{/if}}
+
   <div class="leaderboards">
     {{#if model.leaderboards}}
-      {{#each sortedLeaderboards as |leaderboard|}}
-        <div>
-          <div>
-            <div>
-              {{#link-to "gamificationLeaderboard.byName" leaderboard.id}}
-                {{leaderboard.name}}
-              {{/link-to}}
-            </div>
-          </div>
-          <div>
-            <div>
-              {{d-button
-                class="btn"
-                icon="pencil-alt"
-                title="gamification.edit"
-                action=(action (mut selectedLeaderboardId) leaderboard.id)
-              }}
-              {{d-button
-                class="btn"
-                icon="trash-alt"
-                title="gamification.delete"
-                action=(action "destroyLeaderboard" leaderboard)
-              }}
-            </div>
-          </div>
-        </div>
-      {{/each}}
+      <table>
+        <thead>
+          <th>Name</th>
+          <th>Period</th>
+          <th>Contenders</th>
+          <th>Visible to</th>
+          <th></th>
+          <th></th>
+        </thead>
+
+        <tbody>
+          {{#each sortedLeaderboards as |leaderboard|}}
+            <tr>
+              <td>
+                {{#link-to "gamificationLeaderboard.byName" leaderboard.id}}
+                  {{leaderboard.name}}
+                {{/link-to}}
+              </td>
+              <td>
+                {{#if leaderboard.from_date}}
+                  {{format-date leaderboard.from_date}}
+                  -
+                  {{format-date leaderboard.to_date}}
+                {{/if}}
+              </td>
+              {{!-- <td>placeholder</td>
+              <td>placeholder</td> --}}
+              <td>
+                {{d-button
+                  class="btn-flat"
+                  icon="pencil-alt"
+                  title="gamification.edit"
+                  action=(action (mut selectedLeaderboardId) leaderboard.id)
+                }}
+              </td>
+              <td>
+                {{d-button
+                  class="btn-flat"
+                  icon="trash-alt"
+                  title="gamification.delete"
+                  action=(action "destroyLeaderboard" leaderboard)
+                }}
+              </td>
+            </tr>
+          {{/each}}
+        </tbody>
+      </table>
     {{else}}
-      {{i18n "gamification.leaderboard.none"}}
+      {{#unless creatingNew}}
+        {{i18n "gamification.leaderboard.none"}}
+        {{d-button
+          label="gamification.leaderboard.cta"
+          title="gamification.leaderboard.cta"
```

Comment:
I'm not sure it's useful to have identical label and title, screen readers will read the same text twice.

Reply:
I just copied what the other buttons did tbh. Wasn't sure about changing it.

Reply:
yes it's a mistake we made in the past, generally label alone is enough, unless you need more descriptive title, eg:

label: Add
title: Add Charlie to the designers group